src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -1,50 +1,57 @@
 import React, { useEffect, useMemo, useState } from "react";
 import { Dropdown as ReactBootstrapDropdown } from "react-bootstrap";
 import { FieldError, useForm } from "react-hook-form";
+import { omit } from "lodash";
 
 import { SPACE_PORTALS_ICONS_MAPPING } from "settings";
 
-import { PortalTemplate } from "types/venues";
+import { AnyVenue, PortalTemplate, VenueTemplate } from "types/venues";
+
+import { WithId } from "utils/id";
 
 import { Dropdown } from "components/atoms/Dropdown";
 
 import "./SpacesDropdown.scss";
 
-type SpaceProperty = { template?: PortalTemplate; name: string };
+type SpacesDropdownPortal = { template?: PortalTemplate; name: string };
 export interface SpacesDropdownProps {
-  defaultSpace?: SpaceProperty;
+  parentSpace?: SpacesDropdownPortal;
   setValue: <T>(prop: string, value: T, validate: boolean) => void;
   register: ReturnType<typeof useForm>["register"];
   fieldName: string;
   error?: FieldError;
-  venueSpaces: SpaceProperty[];
+  portals: Record<string, WithId<AnyVenue>>;
 }
 
 export const SpacesDropdown: React.FC<SpacesDropdownProps> = ({
-  defaultSpace,
-  venueSpaces,
+  parentSpace,
+  portals,
   setValue,
   register,
   fieldName,
   error,
 }) => {
-  const [spaceValue, setSpaceValue] = useState(defaultSpace);
+  const [selected, setSelected] = useState(parentSpace);
+
+  // @debt: Probably need to omit returning playa from the useOwnedVenues as it's deprecated and
+  // doesn't exist on SPACE_PORTALS_ICONS_MAPPING
+  const filteredPortals = omit(portals, VenueTemplate.playa);
 
   useEffect(() => {
-    if (defaultSpace) {
-      setSpaceValue(defaultSpace);
+    if (parentSpace) {
+      setSelected(parentSpace);
     }
-  }, [defaultSpace]);
+  }, [parentSpace]);
```

Comment:
Will this make it impossible to un-select parent space? Is that the idea behind `parentSpace`?

Reply:
having `parentSpace` ensures that we have that initial value of the `SpacesDropdown`
And this `useEffect` ensures that we've set the value even if the initial `parentSpace` prop value passed into this component was falsy 
Just test it without the `useEffect` and had numerous issues with initial value for this dropdown